tests/e2e/pageobjects/dashboard/Workspaces.ts

```diff
@@ -104,8 +103,15 @@ export class Workspaces {
 
     async clickActionsStopWorkspaceButton(workspaceName: string) {
         Logger.debug(`Workspaces.clickActionsStopWorkspaceButton for the '${workspaceName}' list item`);
-
-        await this.driverHelper.waitAndClick(this.getActionsPopupButtonLocator(workspaceName, 'Stop Workspace'));
+        // workaround because of issue CRW-3649
+        try {
+            await this.driverHelper.waitAndClick(this.getActionsPopupButtonLocator(workspaceName, 'Stop Workspace'));
+        } catch (e) {
+            Logger.warn(`Workspaces.clickActionsStopWorkspaceButton for the '${workspaceName}' list item - popup was missed, try to click one more time.`);
```

Comment:
IMHO, it would help to understand what was going on if mention that it is workaround to issue CRW-3649 in test output.

Reply:
sure, added

Reply:
thanks